tests/integrations/google_ai_studio/google_ai_studio_test.py

```diff
@@ -119,3 +119,34 @@ async def test_content_generation_async(client):
     # TODO: Re-enable after dictify is fixed
     # assert_correct_output_shape(call.output)
     # assert_correct_summary(call.summary, trace_name)
+
+
+@pytest.mark.retry(max_attempts=5)
+@pytest.mark.skip_clickhouse_client
+def test_send_message(client):
+    import google.generativeai as genai
+
+    genai.configure(api_key=os.getenv("GOOGLE_GENAI_KEY"))
+    model = genai.GenerativeModel(model_name="gemini-1.5-pro", tools="code_execution")
+    chat = model.start_chat()
+    chat.send_message(
+        (
+            "What is the sum of the first 50 prime numbers? "
+            "Generate and run code for the calculation, and make sure you get all 50."
+        )
+    )
+
+    calls = list(client.calls())
+    assert len(calls) == 2
```

Comment:
This is surprising.  Where is the 2nd call coming from?

Reply:
This is because `send_message` is using `GenerativeModel.generate_content` under the hood which we're already patching.

Reply:
Thanks, I would add that comment